analyzers/src/SonarAnalyzer.CSharp/Rules/SqlKeywordsDelimitedBySpace.cs

```diff
@@ -149,25 +161,25 @@ public override void VisitBinaryExpression(BinaryExpressionSyntax node)
                 }
                 else
                 {
-                    base.Visit(node.Left);
-                    base.Visit(node.Right);
+                    Visit(node.Left);
+                    Visit(node.Right);
                 }
             }
 
             private void CheckSpaceBetweenStrings(List<StringWrapper> stringWrappers)
             {
-                for (var i = 0; i < stringWrappers.Count -1; i++)
+                for (var i = 0; i < stringWrappers.Count - 1; i++)
                 {
                     var firstStringText = stringWrappers[i].Text;
                     var secondString = stringWrappers[i + 1];
                     var secondStringText = secondString.Text;
-                    if (firstStringText.Length > 0 &&
-                        IsAlphaNumericOrAt(firstStringText.ToCharArray().Last()) &&
-                        secondStringText.Length > 0 &&
-                        IsAlphaNumericOrAt(secondStringText[0]))
+                    if (firstStringText.Length > 0
+                        && IsAlphaNumericOrAt(firstStringText.ToCharArray().Last())
+                        && secondStringText.Length > 0
+                        && IsAlphaNumericOrAt(secondStringText[0]))
```

Comment:
```suggestion
                    if (firstStringText.Length > 0
                        && IsAlphaNumericOrAt(firstStringText.Last())
                        && secondStringText.Length > 0
                        && IsAlphaNumericOrAt(secondStringText.First()))
```

Reply:
From the suggestion I don't really get it what needs to be changed. Could you please clarify?

Reply:
Just visual simplification.
Instead of `firstStringText.ToCharArray().Last()` -> `firstStringText.Last()`
and similar change to line 179.
Unless I miss something, `firstStringText.Last()` and `secondStringText.First())` retrieve the last and first char respectively.